src/functions/hello/index.ts

```diff
@@ -0,0 +1,18 @@
+import { AWS } from "@serverless/typescript";
+import { handlerPath } from "../../utils.js";
```

Comment:
for example here, I'm not very fond of using relative path ... I tend then to use ts path allowing something like `@/utils` ... unfortunately this isn't supported afaik with `module: "NodeNext"`